.github/workflows/checks-integration.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+  pull_request: {}
```

Comment:
I'm not sure why `checks-integration` has not been triggered on pull request. Perhaps it was just forgotten at https://github.com/optuna/optuna/pull/3580

Reply:
This is slow to run for PR and hard to maintain `Successful` status, so we decided to eliminate it at that time.

Reply:
I see. That sounds reasonable. Thank you for letting me know.